sources/packages/backend/apps/db-migrations/src/sql/Reports/Create-data-inventory-report.sql

```diff
@@ -0,0 +1,507 @@
+INSERT INTO
+    sims.report_configs (report_name, report_sql)
+VALUES
+    (
+        'Data_Inventory_Report',
+        'select
+            -- Personal Info
+            users.first_name as "First Name",
+            users.last_name as "Last Name",
+            students.birth_date as "Date of Birth",
+            students.gender as "Gender",
+            sin_validations.sin as "SIN",
+            users.email as "Email address",
+            students.contact_info -> ''address'' ->> ''addressLine1'' as "Address Line 1",
+            students.contact_info -> ''address'' ->> ''addressLine2'' as "Address Line 2",
+            students.contact_info -> ''address'' ->> ''city'' as "Contact City",
+            students.contact_info -> ''address'' ->> ''provinceState'' as "Contact Province",
+            students.contact_info -> ''address'' ->> ''postalCode'' as "Contact Postal/Zip Code",
+            students.contact_info -> ''address'' ->> ''country'' as "Contact Country",
+            students.contact_info ->> ''phone'' as "Contact Phone Number",
+            -- Account Info
+            (
+                select
+                    case
+                        when exists (
+                            select
+                                1
+                            from
+                                sims.student_restrictions active_student_restrictions
+                            where
+                                active_student_restrictions.student_id = students.id
+                                and active_student_restrictions.is_active = true
+                        ) then ''Y''
+                        else ''N''
+                    end
+            ) as "Active Restrictions Flag",
+            (
+                select
+                    case
+                        when exists (
+                            select
+                                1
+                            from
+                                sims.student_restrictions active_student_restrictions
+                            where
+                                active_student_restrictions.student_id = students.id
+                                and active_student_restrictions.is_active <> true
+                        ) then ''Y''
+                        else ''N''
+                    end
+            ) as "Previous (inactive) Restrictions Flag",
+            (
+                select
+                    SUM (disbursement_overawards.overaward_value)
+                from
+                    sims.disbursement_overawards disbursement_overawards
+                where
+                    disbursement_overawards.student_id = students.id
+                    and disbursement_overawards.disbursement_value_code = ''CSLF''
+                GROUP by
+                    disbursement_overawards.student_id,
+                    disbursement_overawards.disbursement_value_code
+            ) as "CSL Overawards Balance (FT)",
```

Comment:
Can the overawards be converted to a join query, similar to `parent1` and parent `2`?
If not part of the PR we can explore the option in the upcoming ticket.

Reply:
It's possible. I'm going to add that to the upcoming ticket.

Reply:
Added that to #2580